### ISSUE TITLE
Graphical view of locBLAST is not working in Ubuntu

### ISSUE THREAD

**Issue by akshaybioinfo** [AshokHub/locBLAST#6]
I configured locBLAST in UBUNTU server  with address http://14.139.224.59/BpGDB-BLAST/. The text view is working fine but graphical view is not working. While selecting graphical view it shows blank result page. Please suggest the modifications.

**Comment by AshokHub** [comment]
Set permission chmod 777 to the PHP file.

**Comment by AshokHub** [comment]
Grant permission to the locBLAST folder using sudo chmod -R 777 locBLAST command.

**Comment by akshaybioinfo** [comment]
I have already given full permission of 777 to locBLAST as well as all php
and NCBI blast files. But did not get the graphical view, while text view
is working fine.

Thank you.


*With Regards.*
Akshay Singh PhD.
*Research Associate*
*NBPGR, New Delhi -110012*

**Comment by akshaybioinfo** [comment]
The graphical view output of the configured locBLAST server is not working, while text view is working. My locBLAST address is  http://14.139.224.59/BpGDB-BLAST/. I have given full permission of 777 to the locBLAST directory and all the php and blast files as suggested by you previously, but problem is not resolved yet. I am facing the same problem, kindly let me know what changes are required for the graphical view of the blast outputs.
 Thank You.

**Comment by AshokHub** [comment]
Check whether the XML file is created while running the page in the Linux
server. I don't have Linux OS in my system, so I can't verify.

**Comment by akshaybioinfo** [comment]
yes XML file generated while running this page with lock symbol.
this is the structure and permission of the directory. Please check

drwxrwxrwx 5 root     root        12288 Jun 19 10:26 .
drwxr-xr-x 7 root     root         4096 Jun 11 15:42 ..
-rw-r--r-- 1 www-data www-data      338 Jun 19 10:24 blast2024.06.19.10.24.25.fas
-rw-r--r-- 1 www-data www-data     3870 Jun 19 10:24 blast2024.06.19.10.24.25.xml
-rw-r--r-- 1 www-data www-data      338 Jun 19 10:24 blast2024.06.19.10.24.46.fas
-rw-r--r-- 1 www-data www-data     3870 Jun 19 10:24 blast2024.06.19.10.24.46.xml
-rw-r--r-- 1 www-data www-data      338 Jun 19 10:26 blast2024.06.19.10.26.29.fas
-rw-r--r-- 1 www-data www-data     3870 Jun 19 10:26 blast2024.06.19.10.26.29.xml
-rwxrwxrwx 1 root     root     26920560 Jun 11 10:39 blastdb_aliastool
-rwxrwxrwx 1 root     root     28680304 Jun 11 10:40 blastdbcheck
-rwxrwxrwx 1 root     root     36085616 Jun 11 10:40 blastdbcmd
-rwxrwxrwx 1 root     root     40512880 Jun 11 10:40 blast_formatter
-rwxrwxrwx 1 root     root     40591216 Jun 11 10:40 blastn
-rwxrwxrwx 1 root     root        13244 Jun 11 10:23 blastn.php
-rwxrwxrwx 1 root     root     40583024 Jun 11 10:40 blastp
-rwxrwxrwx 1 root     root        12020 Jun 11 10:23 blastp.php
-rwxrwxrwx 1 root     root     40583024 Jun 11 10:40 blastx
-rwxrwxrwx 1 root     root        12556 Jun 11 10:23 blastx.php
-rwxrwxrwx 1 root     root         6243 Jun 11 10:41 cleanup-blastdb-volumes.py
-rwxrwxrwx 1 root     root     28967976 Jun 11 10:41 convert2blastmask
drwxrwxrwx 3 root     root         4096 Jun 11 11:47 db
-rwxrwxrwx 1 root     root     40664944 Jun 11 10:41 deltablast
-rwxrwxrwx 1 root     root        12654 Jun 11 10:23 deltablast.php
-rwxrwxrwx 1 root     root         1135 Jun 11 10:34 download.php
-rwxrwxrwx 1 root     root     27406960 Jun 11 10:41 dustmasker
-rwxrwxrwx 1 root     root        37399 Jun 11 10:23 footer.php
-rwxrwxrwx 1 root     root         4752 Jun 11 10:41 get_species_taxids.sh
-rwxrwxrwx 1 root     root          347 Jun 11 10:23 header.php
-rwxrwxrwx 1 root     root          789 Jun 11 10:23 help.php
drwxrwxrwx 2 root     root         4096 Jun 11 10:23 images
-rwxrwxrwx 1 root     root        38414 Jun 11 10:50 index.php
-rwxrwxrwx 1 root     root        51346 Jun 11 10:41 legacy_blast.pl
-rwxrwxrwx 1 root     root        35147 Jun 11 10:36 LICENSE
-rwxrwxrwx 1 root     root     31243888 Jun 11 10:42 makeblastdb
-rwxrwxrwx 1 root     root     28452720 Jun 11 10:42 makembindex
-rwxrwxrwx 1 root     root     30556784 Jun 11 10:42 makeprofiledb
drwxrwxrwx 2 root     root         4096 Jun 11 10:23 paper
-rwxrwxrwx 1 root     root        27945 Jun 11 10:23 parameters.php
-rwxrwxrwx 1 root     root     40558192 Jun 11 10:42 psiblast
-rwxrwxrwx 1 root     root        13730 Jun 11 10:23 psiblast.php
-rwxrwxrwx 1 root     root        11558 Jun 11 10:36 README.md
-rwxrwxrwx 1 root     root     40632432 Jun 11 10:42 rpsblast
-rwxrwxrwx 1 root     root         9767 Jun 11 10:23 rpsblast.php
-rwxrwxrwx 1 root     root     40644720 Jun 11 10:42 rpstblastn
-rwxrwxrwx 1 root     root         9496 Jun 11 10:23 rpstblastn.php
-rwxrwxrwx 1 root     root     28108144 Jun 11 10:43 segmasker
-rwxrwxrwx 1 root     root        15440 Jun 11 10:25 style.css
-rwxrwxrwx 1 root     root     40619888 Jun 11 10:43 tblastn
-rwxrwxrwx 1 root     root        12677 Jun 11 10:23 tblastn.php
-rwxrwxrwx 1 root     root     40492656 Jun 11 10:43 tblastx
-rwxrwxrwx 1 root     root        11529 Jun 11 10:23 tblastx.php
-rwxrwxrwx 1 root     root        25985 Jun 11 10:43 update_blastdb.pl
-rwxrwxrwx 1 root     root         1191 Jun 11 10:34 update.php
-rwxrwxrwx 1 root     root     32020336 Jun 11 10:43 windowmasker
-rwxrwxrwx 1 root     root        40842 Jun 11 10:23 xml.php

And suggest me the necessary changes.

**Comment by akshaybioinfo** [comment]
while running blast in text view mode the XML files generated and disappears after result page display and in graphical view mode XML files will not disappears, XML files will be available in the directory.

**Comment by AshokHub** [comment]
Check the file size of the XML file.

**Comment by akshaybioinfo** [comment]
The size of XML file is 3.9 KB and another file name with extension .fas is 338 bytes.

**Comment by AshokHub** [comment]
Do the following
1. Copy Linux executable BLAST+ to the server
2. Grant 777 permission to the directory
